langchain/src/chat_models/base.ts

```diff
@@ -199,32 +199,45 @@ export abstract class BaseChatModel<
     }
   }
 
-  /**
-   * Generates chat based on the input messages.
-   * @param messages An array of arrays of BaseMessage instances.
-   * @param options The call options or an array of stop sequences.
-   * @param callbacks The callbacks for the language model.
-   * @returns A Promise that resolves to an LLMResult.
-   */
-  async generate(
+  _flattenLLMResult(llmResult: LLMResult): LLMResult[] {
```

Comment:
I don't think this is actually getting called?

Reply:
Whoops, good catch forgot to delete. Pushed up a commit to remove